### ISSUE TITLE
Build problem for 1.5.2

### ISSUE THREAD

**Issue by berkus** [cococonscious/koji#72]
**Describe the bug**
Fails to build.

**To Reproduce**
```
cargo +nightly  install-update koji
    Updating registry 'https://github.com/rust-lang/crates.io-index'

Package  Installed  Latest  Needs update
koji     v1.5.1     v1.5.2  Yes

Updating koji
```
...snip...
```
   Compiling cocogitto v5.2.0
   Compiling koji v1.5.2
error[E0061]: this function takes 7 arguments but 6 arguments were supplied
   --> /Users/berkus/.cargo/registry/src/github.com-1ecc6299db9ec823/koji-1.5.2/src/lib/commit.rs:43:15
    |
43  |     cocogitto.conventional_commit(&commit_type, scope, summary, body, None, is_breaking_change)?;
    |               ^^^^^^^^^^^^^^^^^^^-------------------------------------------------------------- an argument of type `bool` is missing
    |
note: associated function defined here
   --> /Users/berkus/.cargo/registry/src/github.com-1ecc6299db9ec823/cocogitto-5.2.0/src/lib.rs:364:12
    |
364 |     pub fn conventional_commit(
    |            ^^^^^^^^^^^^^^^^^^^
help: provide the argument
    |
43  |     cocogitto.conventional_commit(&commit_type, scope, summary, body, None, is_breaking_change, /* bool */)?;
    |                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Expected behavior**
Builds.

**Screenshots**
N/A

**Additional context**
N/A

**Comment by its-danny** [comment]
@berkus the issue looks specific to nightly. it's trying to use `cocogitto 5.2.0` on nightly even though I have it set to `5.1.0` in `Cargo.toml` and that's the version used when installing it on stable. not a clue why it's different between the 2.

I think I can fix it by just setting cocogitto to the newest version, but yeah, no idea why it's breaking.

**Comment by its-danny** [comment]
@berkus seems to be fixed in `1.5.3`, tested on stable and nightly. let me know if you're still having issues.

**Comment by berkus** [comment]
It worked! Cheers ❤️
